Framework/PythonInterface/plugins/algorithms/ComputeCalibrationCoefVan.py

```diff
@@ -85,21 +92,23 @@ def validateInputs(self):
         return issues
 
     def get_temperature(self):
+        """Return the temperature
         """
-        tries to get temperature from the sample logs
-        in the case of fail, default value is returned
-        """
+        if not self.getProperty("Temperature").isDefault:
+            return self.getProperty("Temperature").value
         run = self.vanaws.getRun()
         if not run.hasProperty('temperature'):
-            self.log().warning("Temperature sample log is not present in " +
+            self.log().warning("No Temperature given and the 'temperature' " +
+                               "sample log is not present in " +
                                self.vanaws.name() +
                                " T=293K is assumed for Debye-Waller factor.")
             return self.defaultT
         try:
             temperature = float(run.getProperty('temperature').value)
```

Comment:
This will only work if the temperature is a single value, if it is a time series this will return a string with "Time  Value" which will fail the conversion to a float.  It might be checking if it is a Time series and then using timeAverageValue()